### PR TITLE
new "php cheat sheet array functions"

### PR DESCRIPTION
**Pull request by rathoreanirudh**
screenshot from 2015-11-24 19 39 09

### CODE REVIEW

**Comment by daxtheduck** [comment]
@rathoreanirudh Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: lib/DDG/Goodie/IsAwesome/rathoreanirudh.pm [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package DDG::Goodie::IsAwesome::rathoreanirudh;
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/rathoreanirudh.pm]: @rathoreanirudh you need to remove these files from the Pull Request. It should only contain the `php.json` file

Comment (rathoreanirudh) [inline lib/DDG/Goodie/IsAwesome/rathoreanirudh.pm]: I created a new pull request from my master branch and it still gives some kind of error in figlet.t which i didn't edit

Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/rathoreanirudh.pm]: The figlet error is unrelated to your PR -- it's a known bug we'll be fixing

**Comment by moollaza** [comment]
Continued in #1810

**Pull request closed without merging**